packages/firebase_crashlytics/lib/src/firebase_crashlytics.dart

```diff
@@ -68,65 +68,23 @@ class Crashlytics {
     }
   }
 
-  void _setValue(String key, dynamic value) {
+  /// Sets a [value] to be associated with a given [key] for your crash data.
+  ///
+  /// The [value] will be converted to a string by calling [toString] on it.
+  void setCustomKey(String key, dynamic value) {
     // Check that only 64 keys are set.
     if (_keys.containsKey(key) || _keys.length <= 64) {
-      _keys[key] = value;
+      _keys[key] = value.toString();
```

Comment:
I'm not sure what the native SDK does but if you were to pass null as the value we'd get an exception here which is probably not what we want.

Reply:
Dart would throw an exception because there is no `toString` method on `null`.

If we want to check for null values, should we throw an error or just skip them?

Reply:
What do we want then?

Reply:
Looking at the Android source code looks like it actually won't accept a null either and will throw an exception (haven't actually tested).

If that is the case, then perhaps an exception is fine, but a bit more [understandable error](https://api.dart.dev/stable/2.8.2/dart-core/ArgumentError/ArgumentError.notNull.html) would be better.

Reply:
Yeah OK